components/webdriver_server/capabilities.rs

```diff
@@ -81,6 +81,41 @@ impl BrowserCapabilities for ServoCapabilities {
     ) -> Result<bool, WebDriverError> {
         todo!()
     }
+
+    fn webauthn_virtual_authenticators(
+        &mut self,
+        _: &serde_json::Map<std::string::String, Value>,
+    ) -> Result<bool, WebDriverError> {
+        todo!()
+    }
+
+    fn webauthn_extension_uvm(
+        &mut self,
+        _: &serde_json::Map<std::string::String, Value>,
+    ) -> Result<bool, WebDriverError> {
+        todo!()
+    }
+
+    fn webauthn_extension_prf(
+        &mut self,
+        _: &serde_json::Map<std::string::String, Value>,
+    ) -> Result<bool, WebDriverError> {
+        todo!()
```

Comment:
Won't this todo cause any test using these functions to panic?

Reply:
We already have `fn web_socket_url()` that has a similar implementation and that doesn't seem to be causing issues so far. So I guess they are not exercised by current tests and we'll have to mark them as failing if that happens.